<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>693</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KN2C SSL v5.00</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/resources/images/logor2.png</normaloff>:/resources/images/logor2.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabHome">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>751</width>
          <height>551</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridRender"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tabLog">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextEdit" name="txtLog"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStat">
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
       <widget class="QLineEdit" name="txtRefreeSpeed">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtVisionSpeed">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtRecordSpeed">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtTime">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtTimeBall">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>70</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtBall">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>241</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabWM">
       <attribute name="title">
        <string>WorldModel</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTextEdit" name="txtWM"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRef">
       <attribute name="title">
        <string>Referee</string>
       </attribute>
       <widget class="QLineEdit" name="txtrefgs">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>321</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>612</width>
          <height>198</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="txtcmgs_1"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_3"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_2"/>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="txtcmgs_4"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_6"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_5"/>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="txtcmgs_8"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_7"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_9"/>
           </item>
          </layout>
         </item>
         <item row="0" column="3">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="txtcmgs_12"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_10"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_11"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLineEdit" name="txtcmgs_14"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_15"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_13"/>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="txtcmgs_16"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_18"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_17"/>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLineEdit" name="txtcmgs_20"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_19"/>
           </item>
           <item>
            <widget class="QLineEdit" name="txtcmgs_21"/>
           </item>
          </layout>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="txtcmgs_22"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
